src/diffusers/models/attention_processor.py

```diff
@@ -294,6 +294,10 @@ def __init__(
             processor = (
                 AttnProcessor2_0() if hasattr(F, "scaled_dot_product_attention") and self.scale_qk else AttnProcessor()
             )
+
```

Comment:
umm I don't think we should change the default attention processor here 
let's keep this logic in SANA:)